// @what3words/api/umd/index.js
(function (factory) {
    if (typeof module === "object" && typeof module.exports === "object") {
        var v = factory(require, exports);
        if (v !== undefined) module.exports = v;
    }
    else if (typeof define === "function" && define.amd) {
        define(["require", "exports", "./utils", "./requests/available-languages", "./requests/autosuggest", "./requests/convert-to-3wa", "./requests/convert-to-coordinates", "./requests/grid-section"], factory);
    }
})(function (require, exports) {
    "use strict";
    Object.defineProperty(exports, "__esModule", { value: true });
    var utils_1 = require("./utils");
    exports.setOptions = utils_1.setOptions;
    var available_languages_1 = require("./requests/available-languages");
    exports.availableLanguages = available_languages_1.availableLanguages;
    var autosuggest_1 = require("./requests/autosuggest");
    exports.autosuggest = autosuggest_1.autosuggest;
    var convert_to_3wa_1 = require("./requests/convert-to-3wa");
    exports.convertTo3wa = convert_to_3wa_1.convertTo3wa;
    exports.convertTo3waGeoJson = convert_to_3wa_1.convertTo3waGeoJson;
    var convert_to_coordinates_1 = require("./requests/convert-to-coordinates");
    exports.convertToCoordinates = convert_to_coordinates_1.convertToCoordinates;
    exports.convertToCoordinatesGeoJson = convert_to_coordinates_1.convertToCoordinatesGeoJson;
    var grid_section_1 = require("./requests/grid-section");
    exports.gridSection = grid_section_1.gridSection;
    exports.gridSectionGeoJson = grid_section_1.gridSectionGeoJson;
});
